spanner/jdbc/src/main/java/com/example/spanner/jdbc/LoadCsvExample.java

```diff
@@ -0,0 +1,283 @@
+            case BYTES:
+              builder.set(columnName).to(Byte.parseByte(recordValue));
+              break;
+            case INT64:
+              builder.set(columnName).to(Integer.parseInt(recordValue));
+              break;
+            case FLOAT64:
+              builder.set(columnName).to(Float.parseFloat(recordValue));
+              break;
+            case BOOL:
+              builder.set(columnName).to(Boolean.parseBoolean(recordValue));
+              break;
+            case NUMERIC:
+              builder.set(columnName).to(Value.numeric(BigDecimal.valueOf(
+                  Double.parseDouble(recordValue))));
+              break;
+            case DATE:
+              builder.set(columnName).to(com.google.cloud.Date.parseDate(recordValue));
+              break;
+            case TIMESTAMP:
+              builder.set(columnName).to(com.google.cloud.Timestamp.parseTimestamp(recordValue));
+              break;
+            default:
+              System.out.print("Invalid Type. This type is not supported.");
+          }
+        }
+      }
+      mutations.add(builder.build());
+    }
+    CloudSpannerJdbcConnection spannerConnection = connection
+        .unwrap(CloudSpannerJdbcConnection.class);
+    spannerConnection.write(mutations);
+    spannerConnection.close();
+    System.out.println("Data successfully written into table.");
+  }
+
+  static void loadCsv() throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";;
+    String instanceId = "my-instance-id";
+    String databaseId = "my-database-id";
+    String tableName = "my-table-name";
+    String filePath = "my-file-path";
+    String[] optFlags = {"my-opt-flag", "my-opt-arg"};
+    loadCsv(projectId, instanceId, databaseId, tableName,
+        filePath, optFlags);
+  }
+
+  static void loadCsv(String projectId, String instanceId, String databaseId,
+      String tableName, String filePath, String[] optFlags) throws Exception {
+
+    // Initialize option flags
+    Options opt = new Options();
+    opt.addOption("h", true, "File Contains Header");
+    opt.addOption("f", true, "Format Type of Input File "
+        + "(EXCEL, POSTGRESQL_CSV, POSTGRESQL_TEXT, DEFAULT)");
+    opt.addOption("n", true, "String Representing Null Value");
+    opt.addOption("d", true, "Character Separating Columns");
+    opt.addOption("e", true, "Character To Escape");
+    CommandLineParser clParser = new DefaultParser();
+    CommandLine cmd = clParser.parse(opt, optFlags);
+
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+
+    // Initialize connection to Cloud Spanner
+    try (
+        Connection conn = DriverManager.getConnection(
+            String.format("jdbc:cloudspanner:/projects/%s/instances/%s/databases/%s",
+            projectId, instanceId, databaseId));
+        Reader in = new FileReader(filePath)
```

Comment:
It may be cleaner not to separately initialize this outside of the CVSParser. The nested tries are harder to follow